Adyen/UI/Form/Items/Text/FormTextInputItemView.swift

```diff
@@ -8,4 +8,25 @@ import Foundation
 
 /// A form view representing a plain text input.
 /// :nodoc:
-public final class FormTextInputItemView: FormTextItemView<FormTextInputItem> {}
+public final class FormTextInputItemView: FormTextItemView<FormTextInputItem> {
+
+    // MARK: - Initializers
+
+    /// Initializes the text item view.
+    /// - Parameter item: The item represented by the view.
+    public required init(item: FormTextInputItem) {
+        super.init(item: item)
+
+        observe(item.$isEnabled) { [weak self] isEnabled in
```

Comment:
Just a thought:
You can make a internal calculated property `isEnabled`, bind item.$isEnabled to `self.isEnabled` and move all this logic to  `didSet() { .. }`